changelogs/unreleased/gh-9930-mode-option.md

```diff
@@ -0,0 +1,4 @@
+## feature/experimental
```

Comment:
From the documentation request:

> Also, if `prefer_ro` or `prefer_rw` mode is requested, the
> `prefer_local` option is ignored. With all other modes, `prefer_local`
> works as expected.

AFAIU, it was solved.

Reply:
Thanks, fixed.